src/main/java/io/pravega/connectors/flink/AbstractReaderBuilder.java

```diff
@@ -137,6 +139,24 @@ protected PravegaConfig getPravegaConfig() {
                 .collect(Collectors.toList());
     }
 
+    /**
+     * enable/disable pravega reader metrics (default: enabled).
+     *
+     * @param enable boolean
+     * @return A builder to configure and create a reader.
+     */
+    public B enableMetrics(boolean enable) {
```

Comment:
Kafka connector allows the metrics to be configurable and I followed the same here. I don't see any harm in keeping the option since metric is enabled by default?

Reply:
for another reference, this is what we do in Pravega atm:

https://github.com/pravega/pravega/blob/master/config/config.properties#L135